src/iuse_actor.cpp

```diff
@@ -1103,7 +1106,8 @@ static ret_val<tripoint> check_deploy_square( Character *p, item &it, const trip
                 here.mop_spills( tripoint_bub_ms( pnt ) );
                 p->add_msg_if_player( m_info, _( "You mopped up the spill with a nearby mop when deploying a %s." ),
                                       it.tname() );
-                p->moves -= 15;
+                // Note this used to be 15 moves... please mop responsibly, not at the speed of light
```

Comment:
```suggestion
```
This is usefull in the context of the PR but we don't need to keep track of this in the code